### ISSUE TITLE
[studio-ui] Make Sidebar Widget icon support open/close and color configuration

### ISSUE THREAD

**Issue by sumerjabri** [craftercms/craftercms#4351]
Make Sidebar Widget icons support:
* Open/close icon definition (like the old Sidebar)
* Color (like the old Sidebar)
* Consider making the entire widget have a background color to help with usability/visibility/ease of use

**Comment by rart** [comment]
@russdanner need a sample config that has all these features please.

**Comment by sumerjabri** [comment]
@rart https://docs.craftercms.org/en/3.1/site-administrators/studio/sidebar-configuration.html?highlight=open%20close#sample

**Comment by sumerjabri** [comment]
@rart is this still relevant post `ui.xml`? If not, we may need to update the docs and kill this. Let me know.

**Comment by rart** [comment]
Yes, it's still supported. Via `expandedStyle`/`collapsedStyle` of the `icon` and `container` props. For example:

```xml
<icon id="@material-ui/icons/DescriptionOutlined">
    <expandedStyle>
        <color>red</color>
    </expandedStyle>
    <collapsedStyle>
        <color>blue</color>
    </collapsedStyle>
</icon>
<container>
    <expandedStyle>
        <backgroundColor>purple</backgroundColor>
    </expandedStyle>
    <collapsedStyle>
        <backgroundColor>orange</backgroundColor>
    </collapsedStyle>
</container>
```

Children of `expandedStyle` & `collapsedStyle` should be standard css properties.
